python/lsst/pipe/tasks/hips.py

```diff
@@ -454,8 +490,10 @@ def build_quantum_graph(
         registry,
         constraint_order,
         constraint_ranges,
+        output_run,
         where=None,
         collections=None,
+        metadata=None,
```

Comment:
Same comment as I made in lsst/drp_analysis#62 about the possibility for output_run parameter to disagree with value in metadata.